v3/src/transports/http/__tests__/transport.test.ts

```diff
@@ -0,0 +1,163 @@
+import { processHttpRequest, IHttpResponse } from '../transport';
+import { gql } from '../../..';
+// TODO(AS3) Stop depending on this module!
+import { buildSchemaFromSDL } from "apollo-graphql";
+import { GraphQLSchema } from "graphql/type/schema";
+// TODO(AS3) Stop depending on this module!
+import { ForbiddenError } from "apollo-server-core";
+import { GraphQLRequest } from "../../../types";
+
+const testModule = {
+  typeDefs: gql`
+    type Book {
+      title: String
+      author: String
+    }
+
+    type Query {
+      books: [Book]
+      throwForbiddenError: String
+    }
+  `,
+  resolvers: {
+    Query: {
+      books: () => [
+        {
+          title: "Harry Potter and the Chamber of Secrets",
+          author: "J.K. Rowling"
+        },
+        {
+          title: "Jurassic Park",
+          author: "Michael Crichton"
+        },
+      ],
+      throwForbiddenError() {
+        // TODO(AS3) What should we do with these?
+        throw new ForbiddenError("Not allowed!");
```

Comment:
We should chat about these error classes from `apollo-server-errors`.  They are very much a thing from Apollo Server, but I believe current properties of the response from execution make it difficult to use these to decidedly calculate an HTTP status code for a partial success response, which is an important detail to iron out.